docs/designs/conceptual-versioning.md

```diff
@@ -283,16 +283,17 @@ An **ordered** moniker list is provided by moniker definition file restored from
 
 For dynamic, the output path shares the same schema:
 
-`{output-dir}/{locale}?/{group-{monikerListHash}}?/{site-path}`
+`{output-dir}/{siteBasePath}/{monikerListHash}?/{site-path'}`
```

Comment:
`{site-path}` already contains site base path.

Reply:
yes, so I use `site-path'` to represent the path which doesn't contains the site base path, there is a comment blow, any better idea about the name?

---
In reply to: [279677026](https://github.com/dotnet/docfx/pull/4460#discussion_r279677026) [](ancestors = 279677026)